tests/get_maintainer_wrapper_test.sh

```diff
@@ -3,13 +3,14 @@
 . ./src/get_maintainer_wrapper.sh --source-only
 . ./tests/utils --source-only
 
+configurations[get_maintainer_options]="--separator , --nokeywords --nogit --nogit-fallback --norolestats "
```

Comment:
Is this correct? It seems like the test suite doesn't load the default kw configurations. Should get_maintainer_wrapper.sh handle the case where this config is empty?

Reply:
Hi Joao,

You don't need to add this change for this series; if you want to do that, I recommend doing it in another PR.

Anyway, about your question, you can try:
1. In the setup function, you can add:
```
parse_configuration "$KW_CONFIG_SAMPLE"
```
The `parse_configuration` function will parse the kworkflow.config file and populate the configurations hash based on the file information.

2. Add your `get_maintainer_options` in the sample file, for example:
Open -> `tests/samples/kworkflow.config`
Add -> get_maintainer_options="SOMETHING"

3. Update some of the tests and other potential failures.

Notice that the `tests/get_maintainer_wrapper_test.sh` is a little bit outdated when compared with other tests. For example, instead of using `setUp` function, it uses `setupGetMaintainers`.

Reply:
Ok! I reverted the changes referring to the config file.